.github/workflows/hashes.yaml

```
@@ -46,18 +46,18 @@ jobs:
               filename="${the_line[$length]}"
               echo "${filename}"
           }
-          # expects cli files between lines 2-13 and gui 14-18 (comments do not count, 1st line = 0)
-          # to add a new file to the cli, $num must be -gt 1 and -lt 15.
-          # gui $num is now -gt 14 and -lt 20 (new line has been added above)
+          # expects cli files between lines 2-14 and gui 15-19 (comments do not count, 1st line = 0)
+          # to add a new file to the cli, $num must be -gt 1 and -lt 16.
+          # gui $num is now -gt 15 and -lt 21 (new line has been added above)
           # a new gui file will only increase the -lt number by 1
           # changes to extensions / new files must be reflected in the cli_files / gui_files lists below
           num=0
           for line in "${lines[@]}"; do
-              if [ $num -gt 1 ] && [ $num -lt 14 ] ; then
+              if [ $num -gt 1 ] && [ $num -lt 15 ] ; then
                   #CLI
                   filename=$(get_filename "${line}")
                   filenames_cli+=("${filename}")
-              elif [ $num -gt 13 ] && [ $num -lt 20 ] ; then
+              elif [ $num -gt 14 ] && [ $num -lt 20 ] ; then
```

```suggestion
              elif [ $num -gt 14 ] && [ $num -lt 21 ] ; then
```

is this right? 🤔 
ignore me if wrong

---

This was done in previous PR here: https://github.com/monero-project/monero-site/pull/2264/files#diff-e9682b56d580e2645c06a1fdde3c0376d1bee1e1a1cca99a900dbc7f7d902c12L61

Sorry I messed up my history and created an unnecessary conflict. Should be more clear now, please check.